logging/config/cluster-logging-log-forwarding.adoc

```diff
@@ -33,7 +33,8 @@ include::modules/cluster-logging-log-forwarding-about.adoc[leveloffset=+1]
 
 include::modules/cluster-logging-log-forwarding-configure.adoc[leveloffset=+1]
 
-== Additional resources
+include::modules/cluster-logging-log-forwarding-disable.adoc[leveloffset=+1]
 
-Alternatively, you can use Fluentd plugins to forward logs. For more information, see xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[Sending logs to external devices using Fluentd plugins].
+== Additional resources
```

Comment:
s/== Additional resources/.Additional References/

Reply:
@jboxman 
s/== Additional resources/.Additional **resources**/ ??

PS. I hate this one. The Additional resources should be a heading that appears in the TOC, which allows the resource to apply to the entire assembly. As `.Additional reource`, it's not in the TOC and seems to apply to the topic that happens to be last. 

Sigh.